MANIFEST.in

```diff
@@ -0,0 +1,5 @@
+graft ml_pipeline_gen/templates
+graft ml_pipeline_gen/static
+
+prune **/experimental
```

Comment:
`graft **/experimental` for the time being, or if we're confident, pull out `component_lib.py` and `component_spec.yaml`. Please correct me if I'm mistaken here.

Reply:
I'm ignoring experimental to reduce the size of the source distribution. Is there anything in experimental that we currently recommend users use?

Reply:
I think keeping the distribution size small is good, for the CAIP, we do not need anything in experimental.